libraries/networking/src/webrtc/WebRTCDataChannels.cpp

```diff
@@ -4,7 +4,7 @@
 //
 //  Created by David Rowe on 21 May 2021.
 //  Copyright 2021 Vircadia contributors.
-//  Copyright 2021 DigiSomni LLC.
+//  Copyright 2022 DigiSomni LLC.
```

Comment:
This should stay at 2021.

Reply:
Ah, so we stick to earliest date for these?